### PR TITLE
Fix typo in expr example

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: phpcr-shell/querying.rst [hunk 1]
```diff
@@ -196,7 +196,7 @@ Set the value of ``a.title`` to the value of the property :
 
 .. code-block:: bash
 
-    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getProperty('some_property')')
+    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getPropertyValue("some_property")')
```
Comment (dbu) [inline phpcr-shell/querying.rst]: this needs double quotes for sure. does it need to be getPropery or would getPropertyValue (as below) be the correct thing?

Comment (dantleech) [inline phpcr-shell/querying.rst]: Yeah, I think in this case `getPropertyValueWithDefault("some_property", null)` would probably be best.

**Comment by dbu** [comment]
updated

**Comment by dantleech** [comment]
cheers

**Pull request merged**